src/subnet_manager_driver.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
```

Comment:
It's probably best practice to also `#include <vector>` and `<memory>`  here for consistency's sake and to avoid spooky linker errors. Maybe even `spa_communicator.hpp` and `routing_table.hpp`, too.